### PR TITLE
feature flag for turning off reactions in auditorium

### PR DESCRIPTION
**Pull request by dmitrii-zolotukhin**
- [x]  Fixed turning on reactions

fixed sparkletown/internal-sparkle-issues#1170

### CODE REVIEW

File: src/components/templates/Auditorium/components/Section/Section.tsx [hunk 1]
```diff
@@ -96,7 +102,7 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
           <div className={centralScreenClasses}>
             <IFrame containerClassName="Section__iframe" src={iframeUrl} />
             <div className="Section__reactions">
-              {isUserSeated ? (
+              {shouldShowReactions ? (
```
Comment (sunny-viktoryia) [inline src/components/templates/Auditorium/components/Section/Section.tsx:105]: please, check this part.
I've turned off the reactions and took a seat, but the message under the video is still "Welcome! Click on an empty seat to claim it!"

Screenshot 2021-09-15 at 16 56 38

Better behavior would be if there will be just "Welcome!" or no text at all. (you can check what looks better)

**Review by sunny-viktoryia — COMMENTED** [review]
Audience Template:

Reactions are turned on. I enter an audience. And I can see the message "Welcome! ..."
image

Reactions are turned off. I enter an audience. There is no message.

=========================

Auditorium Template

Reactions are turned off. The message "Welcome! Click on an empty seat to claim it!" is still visible although I've already taken a seat.
image

**Comment by dmitrii-zolotukhin** [comment]
> Audience Template:
> 
> Reactions are turned on. I enter an audience. And I can see the message "Welcome! ..."
> image
> 
> Reactions are turned off. I enter an audience. There is no message.
> 
> =========================
> 
> Auditorium Template
> 
> Reactions are turned off. The message "Welcome! Click on an empty seat to claim it!" is still visible although I've already taken a seat.
> image

fixed

**Review by sunny-viktoryia — APPROVED** [review]
Looks good to me. 🙃 
Thanks.

**Pull request merged**